lxd/daemon.go

```diff
@@ -748,7 +748,7 @@ func (d *Daemon) Init() error {
 	d.lxcpath = shared.VarPath("containers")
 
 	/* Make sure all our directories are available */
-	if err := os.MkdirAll(shared.CachePath(), 0700); err != nil {
+	if err := os.MkdirAll(shared.CachePath(), 0711); err != nil {
```

Comment:
No reason, it's just the first mkdir that we call. I can do one for VarPath that's 711 and then do a separate one for CachePath that's 700.